src/node/coverage.ts

```diff
@@ -0,0 +1,267 @@
+// Future goals:
+// - Persist owner/repo names in workspace storage
+// - Cache coverage
+// - Show coverage totals somewhere
+// - Maybe add codecov button to disable and/or view in codecov
+
+import {
+  ExtensionContext,
+  OverviewRulerLane,
+  Position,
+  Range,
+  Uri,
+  commands,
+  window,
+  workspace,
+} from "vscode";
+import axios from "axios";
+
+type Coverage =
```

Comment:
Thoughts on moving these types/consts out into a separate set of files?

Maybe we can have a top level "coverage" directory with all this stuff in separate files within